hack/source-manifests.sh

```diff
@@ -25,9 +25,12 @@ if [ -z "$NOOBAA_IMAGE" ] || [ -z "$ROOK_IMAGE" ]; then
 fi
 
 # always start fresh and remove any previous artifacts that may exist.
+rm -rf $OCS_FINAL_DIR
```

Comment:
I don't really know if this OCS_FINAL_DIR cleanup is entierly related to the change with the bundledmanifests. It seems it should be a separate commit (in this PR).

Reply:
It is related. otherwise the same issue occurs as contents of `bundlemanifests/` are copied inside `OCS_FINAL_DIR` for packaging.

Reply:
Where is this happening? It is not very obvious: before this patch, OCS_FINAL_DIR was not referenced in this script at all...